pkgs/os-specific/darwin/apparency/default.nix

```diff
@@ -0,0 +1,37 @@
+{ lib
+, fetchurl
+, stdenv
+, undmg
+}:
+
+stdenv.mkDerivation {
+  pname = "apparency";
+  version = "1.5.1";
+
+  src = fetchurl {
+    url = "https://web.archive.org/web/20230815073821/https://www.mothersruin.com/software/downloads/Apparency.dmg";
+    sha256 = "sha256-JpaBdlt8kTNFzK/yZVZ+ZFJ3DnPQbogJC7QBmtSVkoQ=";
+  };
+
+  nativeBuildInputs = [ undmg ];
+
+  sourceRoot = "Apparency.app";
+
+  installPhase = ''
+    runHook preInstall
+
+    mkdir -p "$out/Applications/Apparency.app"
+    cp -R . "$out/Applications/Apparency.app"
+
+    runHook postInstall
+  '';
+
+  meta = {
```

Comment:
Please keep the `meta` attributes sorted alphabetically.

Reply:
Updated :+1: